pkg/apis/config/store_test.go

```diff
@@ -51,6 +53,13 @@ func TestStoreLoadWithContext(t *testing.T) {
 		}
 	})
 
+	t.Run("features", func(t *testing.T) {
+		expected, _ := NewFeaturesConfigFromConfigMap(featuresConfig)
+		if diff := cmp.Diff(expected, config.Features, ignoreStuff...); diff != "" {
+			t.Errorf("Unexpected features config (-want, +got): %v", diff)
```

Comment:
```suggestion
			t.Error("Unexpected features config (-want, +got):\n", diff)
```

Reply:
Error adds a space between arguments, but adding `\n` is good, so we still need Errorf but with \n, thanks.